content/intro-to-storybook/vue/en/data.md

```diff
@@ -28,6 +28,7 @@ In a file called `src/store.js` we'll construct a standard Vuex store that respo
 import Vue from 'vue';
 import Vuex from 'vuex';
 
+// src/store.js
```

Comment:
@sshiling bump the comment to the top of the snippet if you don't mind, for consistency in all of the snippets

Reply:
@jonniebigodes done.